internal/compiler/widgets/fluent/std-widgets.slint

```diff
@@ -415,29 +416,29 @@ export StandardListView := ListView {
     }
 }
 
-export ComboBox := FocusScope {
+export ComboBox := Rectangle {
     property <[string]> model;
     property <int> current-index : -1;
     property <string> current-value;
     //property <bool> is-open: false;
-    property<bool> enabled <=> touch.enabled;
+    property<bool> enabled;
+    property<bool> has-focus: fs.has-focus;
```

Comment:
that's less efficient than a two way binding, as the two-way binding will remove the property.  I recommend keeping them as alias.

Reply:
So this effectively merges the two properties? Cool!

I shall use that then:-)

Reply:
yeah, the `<=>` merge the two properties, if it can.

In some case they cannot be merged, if they are both native properties, or if they involve global or different component.
Then it would add some overhead.  But in this case it can be merged.

Reply:
Oh, sorry, I forgot to fix that:-) Was distracted with the gallery failing to build all of a sudden.